Replace references to "traces" with "distributed request". (#71)
w3c/baggage: abstract/common-abstract.md

> Distributed tracing is a set of tools and practices to monitor the health and reliability of a distributed application. A distributed application is an application that consists of multiple components that are deployed and operated separately. It is also known as micro-service.

Reviewer: It seems most of this content is not relevant to baggage, there's really no need to mention distributed tracing.

Reviewer 2: I agree with Yuri. Baggage could be used by any distrubuted system including one that is not traced at all.

Author: Thanks, I have removed this content related to distributed tracing, and made the abstract shorter per the discussion in the last WG meeting.

> Distributed tracing is a set of tools and practices to monitor the health and reliability of a distributed application. A distributed application is an application that consists of multiple components or microservices that are deployed and operated independently.

Reviewer 2: ```suggestion
A distributed application is an application that consists of multiple components, sometimes called microservices, that are deployed and operated independently.

```
